api/src/main/resources/messages.properties

```diff
@@ -93,6 +93,7 @@ error.email.invalid =Invalid Email address
 error.activationkey.invalid =Invalid user activation Key
 error.usernameOrEmail.notNullOrBlank =Username Or email cannot be null or blank
 error.allergyapi.allergy.ReactionNonCoded.cannotBeNumeric=Reaction Non-coded must not be only a number
+error.allergyapi.allergen.nonCodedAllergen.cannotBeNumeric=Non-coded Allergen must not be only a number
```

Comment:
@Muta-Jonathan  looking through your video,do you notice that you are actually not testing the exact changes you added, i actually never see this message when you test with a running instance (on-coded Allergen must not be only a number)

Reply:
Oh sorry @HerbertYiga Let me put an updated one